src/Aspire.Hosting/Kafka/KafkaBuilderExtensions.cs

```diff
@@ -24,9 +24,8 @@ public static IResourceBuilder<KafkaServerResource> AddKafka(this IDistributedAp
         var kafka = new KafkaServerResource(name);
         return builder.AddResource(kafka)
             .WithEndpoint(containerPort: KafkaBrokerPort, hostPort: port, name: KafkaServerResource.PrimaryEndpointName)
-            .WithAnnotation(new ContainerImageAnnotation { Image = "confluentinc/confluent-local", Tag = "7.6.0" })
-            .WithEnvironment(context => ConfigureKafkaContainer(context, kafka))
-            .PublishAsContainer();
```

Comment:
How does the manifest get written now if we don't call `PublishAsContainer()` by default?

Reply:
This is default for container resoures.